configure.ac

```diff
@@ -115,7 +115,7 @@ AC_SUBST([HARDEN_CFLAGS])
 AC_SUBST([HARDEN_LDFLAGS])
 
 # Also check for a few non-hardening-related flags.
-MISC_CXXFLAGS=""
+MISC_CXXFLAGS="-std=c++11"
```

Comment:
I don't have control over this machine.  But sure, I could build a newer version of gcc, but I would very much rather not do that.  If the mosh team wants to drop support for these older compilers, that's fine as long as it is a deliberate decision.  I would hate to lose this support just because of an oversight (so I'm glad we've having this conversation here)